script/Ciel/Day 3/03_01_CIEL03_3_2B.txt

```diff
@@ -407,7 +407,7 @@ I can't go home covered head to toe in blood like this.
 [sha:5cdfe3d38fe6d862edff0ead15f74644666c64f6]{
 -- Page 153, Offset 5241.
 -- もうどこにも―――俺には休んでいい場所なんて、あってはならない。
-There's no place for me anymore――not even a place I can rest.
+There's nowhere for me to go――no place where I may know rest.
```

Comment:
Ending overwraught imo, and "no place for me" at the front works better to convey his othering.